cms/static/js/views/module_edit.js

```diff
@@ -102,7 +102,7 @@
                 };
 
                 return ModuleEdit;
-            })(XBlockView);
+            }(XBlockView));
```

Comment:
Another one here. These are the only changes I see that could possibly change the execution path. Though, I don't remember how javascript evaluates the difference here, maybe they are equivalent.

Reply:
I am 100% sure that I don't know the correct answer to this. A quick google tells me these are `Immediately Invokable Function Expressions`, and [this page](http://www.adequatelygood.com/JavaScript-Module-Pattern-In-Depth.html) uses the "new" format; I still have no idea why.

Tests pass though, so I guess that's one positive?

Reply:
aka IIFEs, pronounced "iffy", which is kind of how I feel about them =). They aren't used much anymore so I'm not too sure about "right" format but I doubt `--fix` would change execution. As far as I can tell, both syntaxes are functionally equivalent -- this quick one-off in the browser console returns the same thing for both:

```
(function(role) {
    return `role: ${role}`;
})('foo');

(function(role) {
    return `role: ${role}`;
}('foo'));
```
The second format does wrap the returned result in parens, but that doesn't change the scope at all.